rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -291,15 +279,24 @@ def load(
         cached_component: Optional["CountVectorsFeaturizer"] = None,
         **kwargs: Any
     ) -> "CountVectorsFeaturizer":
-        from sklearn.feature_extraction.text import CountVectorizer
 
         file_name = meta.get("file")
         featurizer_file = os.path.join(model_dir, file_name)
 
         if os.path.exists(featurizer_file):
             vocabulary = utils.json_unpickle(featurizer_file)
 
-            vectorizer = CountVectorizer()
+            vectorizer = CountVectorizer(
+                token_pattern=meta["token_pattern"],
+                strip_accents=meta["strip_accents"],
```

Comment:
where are these meta properties persisted?

Reply:
`rasa_nlu` persist it separately

Reply:
it is persisted component config